app/models/manageiq/providers/cloud_manager/provision/options_helper.rb

```diff
@@ -15,6 +15,10 @@ def instance_type
     @instance_type ||= Flavor.find_by(:id => get_option(:instance_type))
   end
 
+  def root_disk_size
+    @root_disk_size ||= RootDiskSize.find_by(:id => get_option(:root_disk_size))
```

Comment:
The `RootDiskSize` would need to be a model defined in Rails for this to work.  I am not sure that is what you are trying to do here.  In fact I do not think you need this method at all and can just use `get_option(:root_disk_size)` in the `create_disk` method to get the value.